src/main/java/org/candlepin/subscriptions/capacity/admin/UsageContextSubscriptionProvider.java

```diff
@@ -142,32 +127,16 @@ public Optional<Subscription> getSubscription(
   }
 
   // Filter out subscriptions if we have an exact match on a multipart billingAccountId.
-  private List<Subscription> filterByClosestMatchingMultipartBillingAccountId(
+  private List<Subscription> filterByClosestMatchingBillingAccountId(
```

Comment:
We should actually be ok to just delete this method entirely. The bestMatch logic wont be necessary when its only subscription_id/not mulitpart.

Reply:
I think the need to keep the whole list when there are no matches means we need to keep it to maintain the other behavior. I'll rename it to add clarity and add a comment.

Reply:
They should all be exact matches now. Azure was the only provider requiring us to use a multipart billing account ID like that so we shouldn't run into a situation where there are any to filter out here.

Reply:
Will update